doc/SYNDICATION.md

```diff
@@ -0,0 +1,147 @@
+- **Quick Overview**
```

Comment:
Maybe rather than using bolding for itemising, use .md format for headings? (`#`, `##`, `###`) - that was you can make a index with references (if you wish)